TCAT/AppDelegate.swift

```diff
@@ -106,6 +112,20 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     func applicationWillTerminate(_ application: UIApplication) {
         // Called when the application is about to terminate. Save data if appropriate. See also applicationDidEnterBackground:.
     }
+    
+    func handleShortcut(item: UIApplicationShortcutItem) {
+        let optionsVC = RouteOptionsViewController()
+        if let shortcutData = item.userInfo {
+            let placeData = NSKeyedUnarchiver.unarchiveObject(with: shortcutData["place"] as! Data)
```

Comment:
we should not be forcing `as!` here. Safely unwrap before with `guard let shortcutDataPlace = shortcutData["place"] as? Data else { return }